cmd/down.go

```diff
@@ -27,11 +27,6 @@ func down(cmd *cobra.Command, args []string) {
 	//read the harbor compose file
 	var harborCompose = DeserializeHarborCompose(HarborComposeFile)
 
-	_, _, err := Login()
```

Comment:
I kinda like having `Login()` calls here in the cli layer and keeping the harborAPI code just a thin abstraction of the REST API that isn't aware of the login/logout cli semantics.

Can't the returned `user` and `token` be captured here and passed in to the harborAPI?